ceph-dev-trigger/config/definitions/ceph-dev-trigger.yml

```diff
@@ -80,6 +80,34 @@
                     FORCE=True
                     DISTROS=centos7
                     FLAVOR=notcmalloc
+      - conditional-step:
+          condition-kind: shell
+          condition-command: |
+            echo "${GIT_BRANCH}" | grep -v '\(jewel\|kraken\|luminous\|mimic\|nautilus\)'
+          on-evaluation-failure: dont-run
+          steps:
+            - shell:
+                !include-raw:
+                - ../../../scripts/build_utils.sh
+                - ../../build/notify
+            - trigger-builds:
+                - project: 'ceph-dev'
+                  predefined-parameters: |
+                    BRANCH=${GIT_BRANCH}
+                    FORCE=True
+                    DISTROS=bionic xenial centos7
+                - project: 'ceph-dev'
+                  predefined-parameters: |
+                    BRANCH=${GIT_BRANCH}
+                    FORCE=True
+                    DISTROS=centos7
+                    FLAVOR=notcmalloc
+                - project: 'ceph-dev'
+                  predefined-parameters: |
```

Comment:
this is building crimson always in addition to the other 4 builds right? (for a total of 5)

Reply:
yeah, see https://shaman.ceph.com/builds/ceph/wip-kefu-testing-2019-07-12-1253/59756e32c61c433f7d62ae4dd1c40085f7c8f630/